server/auth/handlers/links.go

```diff
@@ -74,6 +75,7 @@ func GetLinks() Links {
 		RequestPasswordReset:     b + "auth/request-password-reset",
 		PasswordResetRequested:   b + "auth/password-reset-requested",
 		ResetPassword:            b + "auth/reset-password",
+		Invite:                   b + "auth/invite",
```

Comment:
perhaps `auth/accept-invite` would fit better (like (`auth/reset-password` and `auth/logout`)?